apps/test/scratch/scratchProjectTest.js

```diff
@@ -266,7 +266,10 @@ describe('scratch', function() {
   testUtils.setExternalGlobals();
   sinon.stub(console, 'log');
 
-  it('Scratch movement test', function(done) {
+  // Skip test because no current scratch levels in use and was
```

Comment:
should we make a task in Jira to get the new instructions setup working for scratch levels, or are we not worried about having new scratch levels in the future? i'm just worried we'll forget about it

Reply:
I'll make something and put it on the backlog - It didn't sound like in talking to @joshlory  that we thought we were going to make more of these levels anytime soon